Add band colored icons

csadams opened the pull request:
Add shared styles for nodes on different bands (sorry, KML is verbose and duplicative in styles).   Update BAND list & detection code for "No RF" nodes.   Construct styleUrl string and insert into Placemark KML.

csadams committed:
Add band colored icons

Add shared styles for nodes on different bands (sorry, KML is verbose and duplicative in styles).  
Update BAND list & detection code for "No RF" nodes.  
Construct styleUrl string and insert into Placemark KML.

aanon4 commented on writeKML.js:
```diff
@@ -64,16 +171,17 @@ module.exports = {
+            // Get band from 1st digit of frequency field, generate band string & styleUrl
+            const freq1 = node.meshrf && String(node.meshrf.freq)[0] || 'N';
+            const band_name = BAND[freq1] || 'Other';
+            const styleUrl = '#sm_nodes_' + BAND[freq1] || '#sm_nodes'
```
Do you mean
```const styleUrl = '#sm_nodes_' + (BAND[freq1] || '#sm_nodes');```
or something else? Are you trying to provide a default style if BAND[freq1] doesn't exist? (which can happen)

csadams replied:
Yes, the goal is to provide a default if there's no band.  I already detect "None", and set the band to "NoRF"... but now realize that's probably not the right thing to do.  The style URL should be either something like: #sm_nodes_5ghz, OR just #sm_nodes (as a default).  I don't think your parentheses do that, and instead it should be: 
`const styleUrl = ('#sm_nodes_' + BAND[freq1]) || '#sm_nodes';`
That said, I'm going to rework it a bit.  Stand by for updated PR.

csadams replied:
Updated code.  No longer using "NoRF", but rather just "None" in both "band_name" and "as part of the "styleUrl". Added some error detection to catch "Other" situations and use default "#sm_nodes" styleUrl.

csadams committed:
Add band colored icons - fixed

Fix code to add band colored icons.

csadams committed:
Add band colored icons - simplify

Simplified the band colored icon code by removing the step of creating "freq1" and "band_name" objects, and just going straight to the styleUrl.  Removed ExtendedData KML section with band_name field.  Hopefully didn't break anything.

csadams commented:
I made 2 updates to correct the issues you raised and to simplify the code.  Hope I didn't break it in the process.  Let me know if there's a better way to do such things.
